dizoo/multiagent_particle/envs/particle_env.py

```diff
@@ -207,7 +374,8 @@ def step(self, action: list) -> BaseEnvTimestep:
         rew_n = rew_n / (self._max_step * self._n_agent)
         self._sum_reward += rew_n
         occupied_landmarks = info['n'][0][3]
-        if self._step_count >= self._max_step or occupied_landmarks >= self._n_agent or occupied_landmarks >= self._num_landmarks:
+        if self._step_count >= self._max_step or occupied_landmarks >= self._n_agent \
+                or occupied_landmarks >= self._num_landmarks:
```

Comment:
add a new doc file named `modified_predator_result.md` (located in `dizoo/multiagent_particle/`), introduce this modified env and show the difference between QMIX and QTran